_posts/2016-10-31-recurse-center-week-5.md

```diff
@@ -0,0 +1,172 @@
+
+Last week I was chatting with an RCer and watching them coding with Emacs. That obviously got me sidetracked into changing text editors...to Vim!
+
+It made for a bit of a grueling week. I started by looking around at all the options:
+
+- Vanilla Vim?  
+- Neovim?  
+- MacVim?  
+- Vim + Janus?  
+- Vim + An open source config?  
+
+Along with each of those came the question of what plugins to install, what shortcuts to add, etc., etc., etc.
+
+Once I had everything installed, I then needed to dig into the configs so that all my favorite Atom functions worked (multi-cursor, linting, file-explorer, etc.).
+
+Once that was all done, it was time. I could finally...sit at my computer for a week struggling to master the keyboard.
+
+It's been a week though and I'm starting to feel frustrated in Atom. I think that's a good sign 😊.
+
+#### **EpubPress for 🔥🐺**
+
+I finally heard back from Firefox about EpubPress!
+
+It was such a back and forth:
+
+> "Hey, we need you to upload original source files."
+
+I did! I swear! Here's a zip and a link to the github repo. Dig around all you want.
+
+> "Can you please add instructions for building the source files?"
+
+Here are the instructions. But FYI they were also in that repo I sent you...
+
+> "Why am I seeing this error while trying to build?"
+
+Oh no! Because you are running on Windows. Linux plz.
+
+> "The built files don't match the files you submitted."
+
+The built files are all on one line. A single character could be different and the diffs would be off. Is it because I have different versions of the dependencies installed? Gahhh...
+
+Here's a build that isn't optimized.
+
+> "Please submit a build that isn't optimized."
+
+I just did...
+
+> "Oh...we'll look through it ASAP."
+
+KK thx 👍.
+
+So hopefully that gets resolved soon...
+
+#### **P5.js**
+
+Friday night I decided to open up [P5.js](https://p5js.org/) and give it a whirl.
+
+That quickly devolved into being fully sucked in. Over the course of the next day I started experimenting with randomly generated bubbles and easing functions.
+
+Here's a janky .gif of the result:
+
+![bubbles](/assets/posts/bubbles.gif){: .center-image }
+
+Spoke to some other P5 folks around RC and got some tips. Next project: Matrix Code!
+
+### Physical 👊
+
+#### **Meditation**
+
+At the end of last week, a fellow RCer and I began doing brief meditation sessions. It was a very relaxing way to start the day and something I want to keep up.
+
+#### **Dancing**
+
+Always dancing!
+
+A RCer hosted a mini Hip Hop class which was a lot of fun.
+The Halloween party at Founder House had a big dance floor which I also took advantage of 💃.
+
+#### **Cooking**
+
+I checked off the Chicken Parmesan recipe from Food Lab. Made for an excellent week of lunches.
+
+It seems like every day there's some event with pizza...ate pizza for dinner at least 3 times last week. 🍕
+
+### </End>
+
+I feel more aware of the things I want to invest time towards these upcoming weeks.
```

Comment:
Would you elaborate on what these are, or do you feel like you did so earlier in the post?